pkg/inspector/image-inspector.go

```diff
@@ -197,7 +198,8 @@ func (i *defaultImageInspector) createAndExtractImage(client *docker.Client, con
 	}
 
 	reader, writer := io.Pipe()
-	go handleTarStream(reader, i.opts.DstPath)
+	notificationChannel := make(chan int)
+	go handleTarStream(reader, i.opts.DstPath, notificationChannel)
 
 	log.Printf("Extracting image %s to %s", i.opts.Image, i.opts.DstPath)
 	err = client.CopyFromContainer(docker.CopyFromContainerOptions{
```

Comment:
important to note here that since this is a synchronous call we are blocking for the duration of the copy.  So really the only place a timeout would occur is if there is logic in `handleTarStream` or `processTarStream` that executes after this returns